### PR TITLE
@eessex => [XAPP] Correctly set xapp token when refreshed

### PR DESCRIPTION
**Pull request by mzikherman**
This depends on an updated version of https://github.com/artsy/artsy-xapp/pull/7

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -73,7 +73,12 @@ const xappConfig = {
   secret: GRAVITY_SECRET,
 }
 
-xapp.init(xappConfig, bootApp) // eslint-disable-line
+xapp.init(xappConfig, (_unused, token) => {
+  config.GRAVITY_XAPP_TOKEN = token
+  if (hasBooted) return
+  bootApp()
+  hasBooted = true
```
Comment (damassi) [inline index.js]: Why is this necessary?

Comment (mzikherman) [reply index.js]: b/c the callback will happen every time the token is refreshed, but you only want to boot the app once

Comment (mzikherman) [reply index.js]: It's a bit odd. It seems likely that your callback would be some sort non idempotent 'boot' (since you'd want to gate that behind fetching the xapp token).

However, that same callback is what's triggered when the token refreshes (fixed in that PR). In that case, you don't want to re-run your 'boot'.

So perhaps, it makes sense that we only want to callback once, initially ('boot'), and then the refreshes should be handled via some additional mechanism?

Currently the lib [resets its exports](https://github.com/artsy/artsy-xapp/blob/cbc3db68469054ab13bfbffe68edda422a0972bb/index.js#L37), which doesn't seem to be a great way for a client to be able to use it. A client would need to re-require it appropriately, etc.

The fix in the lib is to invoke the callback for every refresh, but with an added guard here.

Comment (damassi) [reply index.js]: Another way to do this would be to use `once`:

```js
import { once } from 'lodash' 

xapp.init(xappConfig, once(_properNameOfVariableThatIsUnused, token) => {
  ... 
}))`
``

**Comment by mzikherman** [comment]
I published the lib under `@artsy/xapp`, bumped that here and will merge.
